modules/end-user-guide/examples/che-ref_a-kubernetes-secret-as-a-file.adoc

```diff
@@ -16,8 +10,10 @@ metadata:
     app.kubernetes.io/component: workspace-secret
 ...
 ----
-+
-Annotations must indicate the given secret is mounted as a file, provide the mount path, and, optionally, specify the name of the container in which the secret is mounted. If there is no target-container annotation, the secret will be mounted into all user containers of the {prod-short} workspace, but this is applicable *_only for the {prod-short} version 7.14_*.
+
+// The corresponding procedure is underdeveloped. Parts of this example will likely end up in the procedure. A comprehensive rewrite is needed. max-cx
```

Comment:
If working on the rewrite is out of scope of this task, file an issue for the work (in GH and a corresponding one in JIRA) and link it here (the GH one), so that the comment would have some follow up.

Reply:
@rkratky, I already opened a JIRA issue, and that's why I've not done more rewriting in this PR - most of the example text here has to be moved to procedures. I will open a GH issue as well.